static/index.html

```diff
@@ -91,6 +91,7 @@ <h3 id="title"></h3>
             }
             else {
                 document.getElementById("interlude").style.visibility = "hidden";
+				document.getElementById("playing").style.visibility = "visible";
```

Comment:
i think you can remove line 90 in this file, the
```js
document.getElementById("playing").style.visibility = "hidden";
```

lmk if it works

Reply:
I changed the `if` statement to check if `state === "playing"` because we don't need any action for the interlude state on page load anymore.
Changing the if statement instead of having an empty action in `state === "interlude"` makes the code more readable

Reply:
clean